### PR TITLE
Code doc update

### LINKED ISSUE #1270: New Doc Proposal:  What Goes In A Pantheon Wordpress Git Repo
We've had a number of outside contractors/new developers working on sites with us over the past month or two who have had trouble with getting their head around the way Pantheon sets up the git repos for Wordpress sites.  Most of them are following the guidelines found here:  https://deliciousbrains.com/storing-wordpress-in-git/ where the git repo only holds the custom WP theme at the root of the repository.

Clearly, this does not work on Pantheon but I find myself repeating the same thing over and over again about how developers need to put the entire Wordpress repo in the Pantheon git repository.  It would be great if there was a doc outlining how a Wordpress (and Drupal) git repo on Pantheon is structured so I have a reference to send to folks.

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1270 
This PR adds the code structure for the git repo and changes the title from "Pantheon Dashboard Code Tool" to "Your Site Code on Pantheon".
@bmackinney Please review

### CODE REVIEW

File: source/docs/articles/sites/code.md [hunk 1]
```diff
@@ -3,12 +3,69 @@ layout: landing
...
+    ├── sites
+        └── all
+           ├── modules
+           └── themes
+        └── default
+           └── settings.php
+    └── themes
+
+<div class="alert alert-info" role="alert">
+<strong>Note</strong>:  The <code>files</code> directory has been omitted. Remember not to include it in your codebase.</div>
```
Comment (bmackinney) [inline source/docs/articles/sites/code.md]: Should indicate this is `sites/default/files`, not just `files`

Comment (bmackinney) [inline source/docs/articles/sites/code.md]: This notice pertains to WordPress code too, with the directory `wp-content/uploads`

File: source/docs/articles/sites/code.md [hunk 2]
```diff
@@ -3,12 +3,69 @@ layout: landing
...
-title: Pantheon Dashboard Code Tool
-description: Learn how to work with your site's code on Pantheon's Dashboard code tool.
+title: Your Site Code on Pantheon
+description: Learn how to work with your site's code on the Pantheon Dashboard.
 keywords: code, commit, sftp, development, how to connect, connection information, wp-admin, admin, administrator, codebase, repository, upstream
 ---
 The Code tool within the Pantheon Dashboard on any environment allows you to interact with your site's code and review the commit log.
 
+## Pantheon Git Repository
+Your codebase is required to import your site into Pantheon, as it is used to create the initial code repository.
```
Comment (bmackinney) [inline source/docs/articles/sites/code.md]: We're not necessarily talking about imports.

File: source/docs/articles/sites/code.md [hunk 3]
```diff
@@ -3,12 +3,69 @@ layout: landing
...
+
+### Drupal Code Structure
+Your archive should include your entire Drupal codebase, including modules, themes, installation profiles, libraries, etc.
```
Comment (bmackinney) [inline source/docs/articles/sites/code.md]: change archive to code directory

**Comment by nataliejeremy** [comment]
@bmackinney Updated--thanks.

**Pull request merged**